spec/index.bs

```diff
@@ -3,8 +3,8 @@ Title: Contact Picker API
 Shortname: contact-picker
 Level: 1
 Status: UD
-Group: wicg
-URL: https://wicg.github.io/contact-api/spec
+Group: webapps
```

Comment:
Is webapps the correct group name here? Not sure with the joint deliverable

Reply:
Yeah, I'm not sure... @tabatkins, sorry to bother you... how do we do a "joint deliverable"?

I couldn't find anything in the docs search for "joint deliverable"... this is a "WebApps" and "DAS" joint. 

https://tabatkins.github.io/bikeshed/#bp-groups

@rayankans, maybe try:
```
Group: webapps
Group: das
```
Or: 
 
```
Group: webapps , das 
```

Reply:
This seems to work:

```
Group: webapps
Group: dap
```

However, it seems to only be related to template generation: https://tabatkins.github.io/bikeshed/#metadata-group

I'll keep it as is for now, it's not exposed anywhere in the spec. I'd like to merge this PR so the links work again.